src/ucp/core/ucp_ep.h

```diff
@@ -12,7 +12,6 @@
 
 #include <uct/api/uct.h>
 #include <ucs/debug/log.h>
-#include <ucs/debug/log.h>
```

Comment:
In future I would suggest to push "cleanups" as a separate PR. It just makes it easier to review the code.